src/document.rs

```diff
@@ -154,14 +154,14 @@ pub trait DocumentOps {
     /// # use ironoxide::prelude::*;
     /// # let sdk: IronOxide = unimplemented!();
     /// # use ironoxide::document::DocumentEncryptOpts;
-    /// let data = "secret data".as_bytes();
+    /// let data = "secret data".to_string().into_bytes();
```

Comment:
again, I think `"secret data".as_bytes().to_owned()` might be slightly better. Most important is consistency in our examples, but just pointing this one out in case the other one changes.